wp-content/mu-plugins/pantheon/assets/images/pantheon-fist-icon-black.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="14px" height="32px" viewBox="0 0 14 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 46.2 (44496) - http://www.bohemiancoding.com/sketch -->
+    <title>Group 11</title>
+    <desc>Created with Sketch.</desc>
```

Comment:
`Group 11` and `Created with Sketch` aren't very accessibility-friendly, they should be replaced with an actual title and description of the image.